### PR TITLE
✨  Sulvo fast fetch implementation

### PR DESCRIPTION
**Pull request by romeoju**
Fast fetch implementation for Sulvo.

### CODE REVIEW

**Comment by careers-sulvo** [comment]
Hi @jridgewell , the PR is now ready for review, all the required checks are passing (except percy but that looks not to be related to this PR changes) ... Let me know if you have any questions or we need to make some changes, thanks in advance :)

**Comment by romeoju** [comment]
Hi @jridgewell , the PR is now ready for review, all the required checks are passing (except percy but that looks not to be related to this PR changes) ... Let me know if you have any questions or we need to make some changes, thanks in advance :)

**Comment by kristoferbaxter** [comment]
Adding @calebcordry and @powerivq to review since this is fast-fetch related.

File: extensions/amp-ad-network-sulvo-impl/0.1/amp-ad-network-sulvo-impl.js [hunk 1]
```diff
@@ -0,0 +1,2016 @@
...
+  'https://securepubads.g.doubleclick.net/gampad/ads';
+
+/** @const {string} */
+const RTC_SUCCESS = '2';
+
+/** @const {string} */
+const DOUBLECLICK_SRA_EXP = 'doubleclickSraExp';
+
+/** @const @enum{string} */
+const DOUBLECLICK_SRA_EXP_BRANCHES = {
```
Comment (calebcordry) [inline extensions/amp-ad-network-sulvo-impl/0.1/amp-ad-network-sulvo-impl.js:125]: I'm not sure I understand the purpose of this implementation. Is this just a clone of `amp-ad-network-doubleclick-impl`? If so I think these will quickly go out of sync.

**Pull request closed without merging**